ocaml/xapi-cli-server/records.ml

```diff
@@ -1494,6 +1494,18 @@ let pool_record rpc session_id pool =
               ~value:(Int64.of_string v)
           )
           ()
+      ; make_field ~name:"license-server"
+          ~get:(fun () -> get_from_map (x ()).API.pool_license_server)
+          ~add_to_map:(fun key value ->
+            Client.Pool.add_to_license_server ~rpc ~session_id ~self:pool ~key
+              ~value
+          )
+          ~remove_from_map:(fun key ->
+            Client.Pool.remove_from_license_server ~rpc ~session_id ~self:pool
+              ~key
+          )
+          ~get_map:(fun () -> (x ()).API.pool_license_server)
+          ()
```

Comment:
I think a `set_in_map` would be useful as well.

Reply:
when `set_in_map` is not set, `set_field` will use `remove_from_map` then `add_to_map` to set map pair. So, command like `xxe pool-param-set uuid=38abeb4c-7a61-7ef2-32a9-35e71cfb0fdf license-server:address=192.168.1.2` is already available.